docker/dev/cassandra-pinot-kafka.yml

```diff
@@ -48,28 +48,20 @@ services:
       JAVA_OPTS: "-Dplugins.dir=/opt/pinot/plugins -Xms4G -Xmx16G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xloggc:gc-pinot-server.log"
     depends_on:
       - pinot-broker
-   kafka:
-    image: docker.io/bitnami/kafka:3.7
-    hostname: kafka
-    container_name: kafka
+  kafka:
+    image: wurstmeister/kafka:latest
```

Comment:
this docker image proved to be not reliable. let's avoid bringing this dependency back and make this work with bitnami provided kafka image.

Reply:
Updated it back, gpt is helpful with updating these configs